db/org_members.go

```diff
@@ -23,52 +24,34 @@ func (ms *MongoStorage) SetOrgMember(salt string, orgMember *OrgMember) (string,
 	ctx, cancel := context.WithTimeout(context.Background(), defaultTimeout)
 	defer cancel()
 
-	if orgMember.OrgAddress.Cmp(common.Address{}) == 0 {
-		return "", ErrInvalidData
-	}
-
```

Comment:
i removed this as redundant since the next check ms.Organization(orgMember.OrgAddress) will anyway return ErrInvalidData if OrgAddress is the zero address